source/sessions/tests/README.rst

```diff
@@ -17,26 +17,25 @@ Several prose tests, which are not easily expressed in YAML, are also presented
 in the Driver Sessions Spec. Those tests will need to be manually implemented
 by each driver.
 
-Snapshot session tests
-======================
-Snapshot sessions tests require server of version 5.0 or higher and
-replica set or a sharded cluster deployment.
-Default snapshot history window on the server is 5 minutes. Running the test in debug mode, or in any other slow configuration
-may lead to `SnapshotTooOld` errors. Drivers can work around this issue by increasing the server's `minSnapshotHistoryWindowInSeconds` parameter, for example:
-
-.. code:: python
-
-    client.admin.command('setParameter', 1, minSnapshotHistoryWindowInSeconds=60)
```

Comment:
AFAICT minSnapshotHistoryWindowInSeconds is unrelated to the ``snapshot`` and ``causalConsistency`` test described below. Could you revert this change? 

Also if the default is 5 minutes, wouldn't 60 seconds be a _decrease_? Wouldn't decreasing the window from 5 minutes to minute make a SnapshotTooOld error more likely?

Reply:
This was under the heading for snapshot sessions, I didn't change the text, so I actually don't know what it is referencing. There was previously a "snapshot sessions" heading that all tests (incorrectly) got grouped under, even though only the first test in the list had to do with snapshot sessions. I think you were the one to add the original note, so I'm happy to defer to your judgment.

Reply:
It's a general remark for all snapshots tests in unified format. In fact it does not apply to the snapshot prose test, as it just configuration validation.
So I think the original version is correct.

This spec was developed before the default [increase to 5 mins](https://jira.mongodb.org/browse/SERVER-47855). Looks like it was also backported to [5.0.0, 5.1.0) versions (C# uses 5.0.14 in EG). In this case we might not longer need this setting, but that needs to be double checked.

Reply:
Ah, so it was intended for the unified tests and not for the prose tests. The heading made it seem like it was a note for the following snapshot prose tests. Thanks for clarifying! The official docs all say that the default is 5 minutes (`=300`) for both 5.0 and 6.0. It seems like perhaps the increase may have originally intended to be `=600` since that is exactly the example used in the [docs](https://www.mongodb.com/docs/v5.0/reference/parameters/#mongodb-parameter-param.minSnapshotHistoryWindowInSeconds). I can update the guidance accordingly and move it up to be with the unified test notes, or I can remove it entirely. Thoughts?

Reply:
Thanks for tracking this down. Keeping at 600 SGTM but it's worth pointing out that we've never needed to use this workaround in pymongo's test suite.

Reply:
I think it's worth keeping this note for debugging purposes.
600 SGTM.  Thanks for catching this.

Reply:
Updated